src/afl-fuzz-init.c

```diff
@@ -1486,7 +1486,7 @@ void check_crash_handling(void) {
       "    sudo launchctl unload -w ${SL}/LaunchDaemons/${PL}.Root.plist\n");
 
 #endif
-  if (!get_afl_env("AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES"))
+  if (!)get_afl_env("AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES")
```

Comment:
This looks ... Odd.

Reply:
Ah. Didn't notice it. I'm sorry. I'll correct it. :sweat_smile: